blockchain/blocks/src/tipset.rs

```diff
@@ -38,6 +37,10 @@ impl TipSetKeys {
         }
         true
     }
+    /// Returns tipset keys
+    pub fn tipset_keys(&self) -> &Vec<Cid> {
```

Comment:
```suggestion
    pub fn tipset_keys(&self) -> &[Cid] {
```
Is there a reason to have it this way?